code/src/App.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react'
+
+//import Artist from './components/Artist';
+import Album from './components/Album';
+import Header from './components/Header';
+
 import data from './data.json'
 
-console.log(data)
+console.log(data);
+
+const albumArray = data.albums.items; 
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <div className="main">
+        <Header />
+        {albumArray.map(item => (
+          <Album key={item.id} albumContent={item} />
+        ))}
+      </div>
+    </>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.